src/sentry/tasks/post_process.py

```diff
@@ -249,18 +270,25 @@ def handle_group_owners(project, group, owners, owner_source):
         pass
 
 
-def update_existing_attachments(event):
+def update_existing_attachments(job):
     """
     Attaches the group_id to all event attachments that were either:
 
     1) ingested prior to the event via the standalone attachment endpoint.
     2) part of a different group before reprocessing started.
     """
-    from sentry.models import EventAttachment
+    # Patch attachments that were ingested on the standalone path.
+    with sentry_sdk.start_span(op="tasks.post_process_group.update_existing_attachments"):
+        try:
```

Comment:
right yeah. this makes a lot of sense. i've been trying to balance optimizing the code vs. making minimal/safe changes so this could go in the next round